src/components/form/examples/props-factory-form.tsx

```diff
@@ -44,7 +44,7 @@ export class PropsFactoryFormExample {
             'limel-example-props-factory-picker'
         ) {
             console.log('propsFactory - subSchema:', subSchema);
-            const properties = { injected: this.somePropertyToInject };
+            const properties = { injectedObject: this.somePropertyToInject };
```

Comment:
I would like to show that props should be camelcased when set by the propsFactory, but I couldn't come up with a better name than `injectedObject`, which might just make people assume we want them to use Hungarian notation… 🤔

Reply:
Maybe one of these will do? :slightly_smiling_face: 
- `injectedData`
- `myData`
- `componentData`

Reply:
⚡